spec/features/user/user_signup_spec.rb

```diff
@@ -19,21 +19,6 @@
     expect(current_path).to eq(user_path(User.find(1)))
   end
 
-  xit "can't sign up with an invalid email" do
-    within(".navbar-nav") do
-      click_link_or_button("Sign up")
-    end
-    within("#signUpModal") do
-      fill_in("name", with: "Paul")
-      fill_in("email", with: "hi")
-      fill_in("password", with: "test")
-      click_link_or_button("Sign up")
-    end
-    expect(User.count).to eq(0)
-    expect(page).to have_content("Input is invalid, please try again")
-    expect(current_path).to eq(users_path)
-  end
-
```

Comment:
Can you do that? Can you place a conditional in a validation? I have been asking that for days?